suite-native/app/e2e/pageObjects/deviceManagerActions.ts

```diff
@@ -12,6 +12,13 @@ class DeviceManagerActions {
         await waitFor(deviceSettingsButton).toBeVisible().withTimeout(30000);
         await deviceSettingsButton.tap();
     }
+
+    async tapOpenPassphraseButton() {
+        const openPassphraseButton = element(by.id('@device-manager/passphrase/add'));
+
+        await waitFor(openPassphraseButton).toBeVisible().withTimeout(2e3);
```

Comment:
I would prefer to keep the timeout syntax consistent with the rest of the calls. Either change it all (in a separate PR) or keep the current approach. Being able to use something such as `30s` would be even better.

Reply:
Updated.